wolvic/browser/session_settings.cc

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "wolvic/browser/session_settings.h"
+
+#include "base/check.h"
+
+namespace content {
```

Comment:
Why is this in content namespace?

Reply:
It seems that we have older parts of our code still in the "content" namespace while new parts are in the  "wolvic" one. I moved newly created files to "wolvic", but probably we should update older files as well at some point.